modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2385,6 +2385,14 @@ export class Wallet implements IWallet {
       throw new Error('txRequestId missing from signed transaction');
     }
 
+    // TODO: Remove conditional when moved to txRequestFull for everything
+    if (this._wallet.type === 'custodial') {
+      this.bitgo
```

Comment:
Missing `await` or `return`? Also do we want to return here or should this also call `sendTxRequest` below?